api/src/certification/results/infrastructure/utils/pdf/templates/certificate.js

```diff
@@ -173,6 +174,21 @@ export default function generateV3AttestationTemplate({ pdf, data, translate })
       .text(globalLevelDescription);
   }
 
+  if (data.acquiredComplementaryCertification) {
+    pdf
+      .font('Nunito-Bold')
+      .fontSize(11)
+      .fillColor('#253858')
+      .text(translate('certification.attestation.v3.complementary-content.title'), 530, 470, {
+        width: 250,
+      });
+
+    pdf.image(path.resolve(__badgesDirname, 'CLEA.png'), 628, 490, {
```

Comment:
Proposition pour éviter les magic string et relier à la notion métier

```suggestion
    pdf.image(path.resolve(__badgesDirname, `${ComplementaryCertificationKeys.CLEA}.png`), 628, 490, {
```